tests/swaps/SwapsExternal.js

```diff
@@ -305,6 +305,8 @@ contract("SwapsExternal", (accounts) => {
 		});
 
 		it("Should be able to withdraw fees", async () => {
+			const maxDisagreement = new BN(wei("15", "ether")); // need to adjust to 15% max slippage for this test cases because the LP is not rebalanced
```

Comment:
can't we just mock LP to return predefined prices & disagreement?

Reply:
as far as i remember, the price is not depending on any liquidity pool, but is static already.

Reply:
Changed the max slippage to 5% and rectified test descriptions